app/routers/roles.py

```
@@ -0,0 +1,94 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlalchemy.orm import Session
+from typing import List
+from app.database import get_db
+from app import crud, schemas
+
+router = APIRouter(prefix="/roles", tags=["roles"])
+
+
+@router.post("/", response_model=schemas.RoleResponse, status_code=status.HTTP_201_CREATED)
+def create_role(role: schemas.RoleCreate, db: Session = Depends(get_db)):
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)